### ISSUE TITLE
Failure with development version of tidyr

### ISSUE THREAD

**Issue by hadley** [shntnu/pertminr#126]
```
1. Failure: `replicate_correlation` measures correlationbetween replicates in
    `%>%`(...) not equal to correlations %>% as.data.frame().
    Attributes: < Component "row.names": Numeric: lengths (2, 3) differ >
    Component "variable": Lengths (2, 3) differ (string compare on first 2)
    Component "variable": 1 string mismatch
    Component "median": Numeric: lengths (2, 3) differ
```
(from <https://win-builder.r-project.org/incoming_pretest/tidyr_0.8.1_20180509_170155/reverseDependencies/summary.txt>)

Any ideas what might cause this? There's only one change to tidyr, which should only make a case that previously threw an error now work:

* `unnest()` weakens test of "atomicity" to restore previous behaviour when
  unnesting factors and dates (#407).

**Comment by shntnu** [comment]
I wasn't able to reproduce that on OSX (session info below) and unfortunately I don't have access to a Windows machine right now. Do you have the revdep summary for any other OS?

I suspect the `doParallel` package might be interfering in some way – that's the only thing I can imagine would behave differently across the OS's. 

<details>

<summary>Session info</summary>

``` r
devtools::session_info()
#> Session info -------------------------------------------------------------
#>  setting  value                       
#>  version  R version 3.4.4 (2018-03-15)
#>  system   x86_64, darwin17.3.0        
#>  ui       unknown                     
#>  language (EN)                        
#>  collate  en_US.UTF-8                 
#>  tz       America/Toronto             
#>  date     2018-05-09
#> Packages -----------------------------------------------------------------
#>  package    * version    date       source                          
#>  assertthat   0.2.0      2017-04-11 CRAN (R 3.4.3)                  
#>  backports    1.1.2      2017-12-13 CRAN (R 3.4.3)                  
#>  base       * 3.4.4      2018-03-15 local                           
#>  bindr        0.1.1      2018-03-13 CRAN (R 3.4.4)                  
#>  bindrcpp     0.2.2      2018-03-29 CRAN (R 3.4.4)                  
#>  broom        0.4.4      2018-03-29 CRAN (R 3.4.4)                  
#>  cellranger   1.1.0      2016-07-27 CRAN (R 3.4.3)                  
#>  cli          1.0.0      2017-11-05 CRAN (R 3.4.3)                  
#>  codetools    0.2-15     2016-10-05 CRAN (R 3.4.4)                  
#>  colorspace   1.3-2      2016-12-14 CRAN (R 3.4.3)                  
#>  compiler     3.4.4      2018-03-15 local                           
#>  crayon       1.3.4      2017-09-16 CRAN (R 3.4.3)                  
#>  cytominer  * 0.1.0.9000 2018-05-05 local                           
#>  datasets   * 3.4.4      2018-03-15 local                           
#>  devtools     1.13.5     2018-02-18 CRAN (R 3.4.4)                  
#>  digest       0.6.15     2018-01-28 cran (@0.6.15)                  
#>  dplyr      * 0.7.4      2017-09-28 CRAN (R 3.4.3)                  
#>  evaluate     0.10.1     2017-06-24 CRAN (R 3.4.3)                  
#>  forcats    * 0.3.0      2018-02-19 CRAN (R 3.4.4)                  
#>  foreach      1.4.4      2017-12-12 CRAN (R 3.4.3)                  
#>  foreign      0.8-70     2018-04-23 CRAN (R 3.4.4)                  
#>  ggplot2    * 2.2.1      2016-12-30 CRAN (R 3.4.3)                  
#>  glue         1.2.0.9000 2018-05-10 Github (tidyverse/glue@7230ed2) 
#>  graphics   * 3.4.4      2018-03-15 local                           
#>  grDevices  * 3.4.4      2018-03-15 local                           
#>  grid         3.4.4      2018-03-15 local                           
#>  gtable       0.2.0      2016-02-26 CRAN (R 3.4.3)                  
#>  haven        1.1.1      2018-01-18 CRAN (R 3.4.3)                  
#>  hms          0.4.2      2018-03-10 CRAN (R 3.4.4)                  
#>  htmltools    0.3.6      2017-04-28 CRAN (R 3.4.3)                  
#>  httr         1.3.1      2017-08-20 CRAN (R 3.4.3)                  
#>  iterators    1.0.9      2017-12-12 CRAN (R 3.4.3)                  
#>  jsonlite     1.5        2017-06-01 CRAN (R 3.4.3)                  
#>  knitr        1.20       2018-02-20 cran (@1.20)                    
#>  lattice      0.20-35    2017-03-25 CRAN (R 3.4.4)                  
#>  lazyeval     0.2.1      2017-10-29 CRAN (R 3.4.3)                  
#>  lubridate    1.7.4      2018-04-11 CRAN (R 3.4.4)                  
#>  magrittr     1.5        2014-11-22 CRAN (R 3.4.3)                  
#>  Matrix       1.2-14     2018-04-09 CRAN (R 3.4.4)                  
#>  memoise      1.1.0      2017-04-21 CRAN (R 3.4.3)                  
#>  methods    * 3.4.4      2018-03-15 local                           
#>  mnormt       1.5-5      2016-10-15 CRAN (R 3.4.3)                  
#>  modelr       0.1.1      2017-07-24 CRAN (R 3.4.3)                  
#>  munsell      0.4.3      2016-02-13 CRAN (R 3.4.3)                  
#>  nlme         3.1-137    2018-04-07 CRAN (R 3.4.4)                  
#>  parallel     3.4.4      2018-03-15 local                           
#>  pillar       1.2.2      2018-04-26 CRAN (R 3.4.4)                  
#>  pkgconfig    2.0.1      2017-03-21 CRAN (R 3.4.3)                  
#>  plyr         1.8.4      2016-06-08 CRAN (R 3.4.3)                  
#>  psych        1.8.3.3    2018-03-30 CRAN (R 3.4.4)                  
#>  purrr      * 0.2.4      2017-10-18 CRAN (R 3.4.3)                  
#>  R6           2.2.2      2017-06-17 CRAN (R 3.4.3)                  
#>  Rcpp         0.12.16    2018-03-13 CRAN (R 3.4.4)                  
#>  readr      * 1.1.1      2017-05-16 CRAN (R 3.4.3)                  
#>  readxl       1.1.0      2018-04-20 CRAN (R 3.4.4)                  
#>  reshape2     1.4.3      2017-12-11 CRAN (R 3.4.3)                  
#>  rlang        0.2.0      2018-02-20 CRAN (R 3.4.4)                  
#>  rmarkdown    1.9        2018-03-01 cran (@1.9)                     
#>  rprojroot    1.3-2      2018-01-03 CRAN (R 3.4.3)                  
#>  rvest        0.3.2      2016-06-17 CRAN (R 3.4.3)                  
#>  scales       0.5.0      2017-08-24 CRAN (R 3.4.3)                  
#>  stats      * 3.4.4      2018-03-15 local                           
#>  stringi      1.2.2      2018-05-02 CRAN (R 3.4.4)                  
#>  stringr    * 1.3.0      2018-02-19 cran (@1.3.0)                   
#>  tibble     * 1.4.2      2018-01-22 cran (@1.4.2)                   
#>  tidyr      * 0.8.0.9000 2018-05-10 Github (tidyverse/tidyr@be7bf93)
#>  tidyverse  * 1.2.1      2017-11-14 CRAN (R 3.4.4)                  
#>  tools        3.4.4      2018-03-15 local                           
#>  utils      * 3.4.4      2018-03-15 local                           
#>  withr        2.1.2      2018-03-15 CRAN (R 3.4.4)                  
#>  xml2         1.2.0      2018-01-24 cran (@1.2.0)                   
#>  yaml         2.1.19     2018-05-01 CRAN (R 3.4.4)
```

</details>

**Comment by shntnu** [comment]
Fixed in #135
